validateWorkerVersions.ps1

```diff
@@ -26,7 +26,7 @@ function removeBomIfExists([string]$data)
     return $data
 }
 
-$cliCsprojPath = "./src/Azure.Functions.Cli/Azure.Functions.Cli.csproj"
+$cliCsprojPath = "$PSScriptRoot/src/Azure.Functions.Cli/Azure.Functions.Cli.csproj"

```

Comment:
Isn't the build machine on windows? How come that wasn't failing?

Reply:
It looks like the `XmlDocument.Save` method converts the `.` component of the .csproj path to be `C:\WINDOWS\system32` for some reason. Strangely enough, other methods like `Set-Location` and `Get-ChildItem` do not seem to have this issue. If the issue resides solely in the `XmlDocument.Save` method, then it would make sense this would run appropriately on ADO since we do not add the `-Update` parameter there, and therefore do not execute `XmlDocument.Save`